cmd/pinniped-proxy/src/cli.rs

```diff
@@ -0,0 +1,19 @@
+use structopt::StructOpt;
```

Comment:
So this PR introduces the structured options crate (package). You can read more about it at https://docs.rs/structopt/0.3.21/structopt/ , but think of it as similar to the golang `cobra` module.